graphs/possible_bipartition.py

```diff
@@ -1,12 +1,56 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(N)
+        Space Complexity: O(NE)
```

Comment:
⏱ The time complexity would actually be O(N + E) since you traverse each node and each edge once.

🪐 The space complexity would be O(N) since you have at most N nodes in either set or in your queue.